ibm_cloud_sdk_core/utils.py

```diff
@@ -218,21 +222,30 @@ def __read_from_vcap_services(service_name: str) -> dict:
         A set of service configuration key-value pairs.
     """
     vcap_services = getenv('VCAP_SERVICES')
+    if not vcap_services:
+        vcap_services_file = getenv('VCAP_SERVICES_FILE')
```

Comment:
let's remove the code that loads the VCAP object from a file.

Reply:
done